packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -576,7 +576,7 @@ export default async function main(
     }
 
     if (deployment.readyState === 'CANCELED') {
-      output.log('The deployment has been canceled.');
+      output.print('The deployment has been canceled.');
```

Comment:
Why this change? It will remove the trailing newline, which I think we don't want to happen.

Reply:
@TooTallNate Yes you are right, I just fixed this by adding a newline at the end, please check.